From 88df28277e770d494edb41f7ce826a670c14d55b Mon Sep 17 00:00:00 2001
From: Zhihang Wei <wzh@ilbers.de>
Date: Wed, 10 Dec 2025 16:17:56 +0100
Subject: [PATCH] Disable extension board output

Extension board output in dtb cause errors when applying the dtso files.
Disable for now.

Signed-off-by: Zhihang Wei <wzh@ilbers.de>

---
 ...yboard-pollux-peb-av-10-etml1010g3dra.dtso | 30 -------------------
 ...hyboard-pollux-peb-av-10-ph128800t006.dtso | 30 -------------------
 2 files changed, 60 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-etml1010g3dra.dtso b/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-etml1010g3dra.dtso
index aceb5b6056ef..73ba0631ac59 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-etml1010g3dra.dtso
+++ b/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-etml1010g3dra.dtso
@@ -10,36 +10,6 @@
 #include <dt-bindings/clock/imx8mp-clock.h>
 #include "imx8mp-phyboard-pollux-peb-av-10.dtsi"
 
-&backlight_lvds0 {
-	brightness-levels = <0 8 16 32 64 128 255>;
-	default-brightness-level = <8>;
-	enable-gpios = <&gpio5 1 GPIO_ACTIVE_HIGH>;
-	num-interpolated-steps = <2>;
-	pwms = <&pwm4 0 50000 0>;
-	status = "okay";
-};
-
-&lcdif2 {
-	status = "okay";
-};
-
-&lvds_bridge {
-	assigned-clocks = <&clk IMX8MP_CLK_MEDIA_LDB>, <&clk IMX8MP_VIDEO_PLL1>;
-	assigned-clock-parents = <&clk IMX8MP_VIDEO_PLL1_OUT>;
-	/*
-	 * The LVDS panel uses 72.4 MHz pixel clock, set IMX8MP_VIDEO_PLL1 to
-	 * 72.4 * 7 = 506.8 MHz so the LDB serializer and LCDIFv3 scanout
-	 * engine can reach accurate pixel clock of exactly 72.4 MHz.
-	 */
-	assigned-clock-rates = <0>, <506800000>;
-	status = "okay";
-};
-
-&panel_lvds0 {
-	compatible = "edt,etml1010g3dra";
-	status = "okay";
-};
-
 &pwm4 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-ph128800t006.dtso b/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-ph128800t006.dtso
index 559286f384be..73ba0631ac59 100644
--- a/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-ph128800t006.dtso
+++ b/arch/arm64/boot/dts/freescale/imx8mp-phyboard-pollux-peb-av-10-ph128800t006.dtso
@@ -10,36 +10,6 @@
 #include <dt-bindings/clock/imx8mp-clock.h>
 #include "imx8mp-phyboard-pollux-peb-av-10.dtsi"
 
-&backlight_lvds0 {
-	brightness-levels = <0 8 16 32 64 128 255>;
-	default-brightness-level = <8>;
-	enable-gpios = <&gpio5 1 GPIO_ACTIVE_HIGH>;
-	num-interpolated-steps = <2>;
-	pwms = <&pwm4 0 66667 0>;
-	status = "okay";
-};
-
-&lcdif2 {
-	status = "okay";
-};
-
-&lvds_bridge {
-	assigned-clocks = <&clk IMX8MP_CLK_MEDIA_LDB>, <&clk IMX8MP_VIDEO_PLL1>;
-	assigned-clock-parents = <&clk IMX8MP_VIDEO_PLL1_OUT>;
-	/*
-	 * The LVDS panel uses 66.5 MHz pixel clock, set IMX8MP_VIDEO_PLL1 to
-	 * 66.5 * 7 = 465.5 MHz so the LDB serializer and LCDIFv3 scanout
-	 * engine can reach accurate pixel clock of exactly 66.5 MHz.
-	 */
-	assigned-clock-rates = <0>, <465500000>;
-	status = "okay";
-};
-
-&panel_lvds0 {
-	compatible = "powertip,ph128800t006-zhc01";
-	status = "okay";
-};
-
 &pwm4 {
 	status = "okay";
 };
-- 
2.39.5

